anndata/_core/anndata.py

```diff
@@ -473,27 +527,20 @@ def _init_as_actual(
                     X = np.array(X, dtype, copy=False)
             # data matrix and shape
             self._X = X
-            self._n_obs, self._n_vars = self._X.shape
+            n_obs, n_vars = self._X.shape
```

Comment:
```suggestion
            n_obs, n_vars = X.shape
```

A little more simple even?